core/auth/src/main/java/software/amazon/awssdk/auth/credentials/internal/ProfileCredentialsUtils.java

```diff
@@ -49,6 +50,8 @@
 public final class ProfileCredentialsUtils {
     private static final String STS_PROFILE_CREDENTIALS_PROVIDER_FACTORY =
         "software.amazon.awssdk.services.sts.internal.StsProfileCredentialsProviderFactory";
+    private static final String SSO_PROFILE_CREDENTIALS_PROVIDER_FACTORY =
+        "software.amazon.awssdk.services.sso.internal.SsoCredentialsProviderFactory";
```

Comment:
I don't actually see this class in this PR. Am I blind?

Reply:
My bad, the name of this class is changed, it should be `SsoProfileCredentialsProviderFactory` now.